src/components/DatePickerInline/Month.tsx

```diff
@@ -151,7 +158,7 @@ export const weekMargin = 6;
 export const weekSize = daySize + weekMargin;
 export const montHeaderHeight = 56;
 export const monthHeaderSingleMarginTop = 4;
-export const monthHeaderSingleMarginBottom = 8 + 44 + 12;
+export const monthHeaderSingleMarginBottom = 8 + 22 + 12;
```

Comment:
does this affect anything for any other component?

Reply:
no. @shashank1010